### PR TITLE
Modules, terrakube, README, oh my!

### PR DESCRIPTION
**Pull request by adrnswanberg**
😎

### CODE REVIEW

File: terraform/aws/kube/kube.tf [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            name  = "BUCKET_QUEUE"
+            value = "sqs://${var.file_metadata_queue_name}"
+          }
+          env {
+            name  = "AWS_REGION"
+            value = var.file_storage_bucket_region
+          }
+          env {
+            name  = "MYSQL"
+            value = "mysql://wandb:wandb_root_password@${var.database_endpoint}/wandb_local"
```
Comment (vanpelt) [inline terraform/aws/kube/kube.tf:119]: We should make the database password a variable with a default.

File: terraform/aws/kube/kube.tf [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  }
+}
+
+##########################################
+# Variables
+##########################################
+
+variable "license" {
+  description = "The license string for your local instance."
+  type        = string
```
Comment (vanpelt) [inline terraform/aws/kube/kube.tf:24]: In my tests on <=0.9.29 this didn't work, if it works now AWESOME.  If not, we could pair for < 15 minutes to get to the bottom of it 😎

**Review by vanpelt — APPROVED** [review]
This is so epic!  Love it.

File: terraform/aws/kube/kube.tf [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+resource "local_file" "kubeconfig" {
+  filename = "kubeconfig.yaml"
+  content  = <<KUBECONFIG
+    apiVersion: v1
+    kind: Config
+    clusters:
+    - cluster:
+        server: ${var.kube_cluster_endpoint}
+        certificate-authority-data: ${var.kube_cert_data}
```
Comment (vanpelt) [inline terraform/aws/kube/kube.tf:182]: Ahh, I saw this and thought it was SSL, this is just for the kubectl client, I see.

File: terraform/aws/kube/kube.tf (hunk 2 above)
Comment (adrnswanberg) [reply terraform/aws/kube/kube.tf:24]: It worked for me but I only tried this version of the stack once. Where did it go wrong for you?

Comment (vanpelt) [reply terraform/aws/kube/kube.tf:24]: If the env tarball doesn't exist in bucket I didn't think it worked on initial boot, but I'm not positive.

File: terraform/aws/kube/kube.tf (hunk 1 above)
Comment (adrnswanberg) [reply terraform/aws/kube/kube.tf:119]: Done

File: terraform/aws/kube/kube.tf (hunk 2 above)
Comment (adrnswanberg) [reply terraform/aws/kube/kube.tf:24]: i did the full apply again and it worked, but im using 0.9.30

**Pull request merged**